### ISSUE TITLE
NPE in EditorMenus.folderMap()

### ISSUE THREAD

**Issue by stephengold** [stephengold/Maud#7]
```text
Oct 18, 2021 2:41:52 PM jme3utilities.nifty.BasicScreenController perform
SEVERE: Caught unexpected throwable:
java.lang.NullPointerException
        at maud.menu.EditorMenus.folderMap(EditorMenus.java:110)
        at maud.menu.BuildMenus.newAssetLocation(BuildMenus.java:496)
        at maud.action.NewAction.testForPrefixes(NewAction.java:207)
        at maud.action.NewAction.process(NewAction.java:172)
        at maud.action.EditorInputMode.onAction(EditorInputMode.java:180)
        at jme3utilities.nifty.BasicScreenController.perform(BasicScreenController.java:242)
        at jme3utilities.nifty.PopupMenu.onEvent(PopupMenu.java:219)
        at jme3utilities.nifty.PopupMenu.onEvent(PopupMenu.java:45)
        at de.lessvoid.nifty.Nifty$ClassSaveEventTopicSubscriber.onEvent(Nifty.java:1766)
        at org.bushe.swing.event.ThreadSafeEventService.publish(ThreadSafeEventService.java:980)
        at org.bushe.swing.event.ThreadSafeEventService.publish(ThreadSafeEventService.java:920)
        at de.lessvoid.nifty.Nifty.publishEvent(Nifty.java:311)
        at de.lessvoid.nifty.controls.menu.MenuControl.activateItem(MenuControl.java:112)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at de.lessvoid.nifty.NiftyMethodInvoker.callMethod(NiftyMethodInvoker.java:157)
        at de.lessvoid.nifty.NiftyMethodInvoker.performInvoke(NiftyMethodInvoker.java:102)
        at de.lessvoid.nifty.Nifty$DelayedMethodInvoke.perform(Nifty.java:1481)
        at de.lessvoid.nifty.Nifty.invokeMethods(Nifty.java:1457)
        at de.lessvoid.nifty.Nifty.handleDynamicElements(Nifty.java:439)
        at de.lessvoid.nifty.Nifty.access$1600(Nifty.java:87)
        at de.lessvoid.nifty.Nifty$NiftyInputConsumerImpl.processEvent(Nifty.java:1727)
        at de.lessvoid.nifty.Nifty$NiftyInputConsumerImpl.processMouseEvent(Nifty.java:1665)
        at com.jme3.niftygui.InputSystemJme.handleMouseEvent(InputSystemJme.java:126)
        at com.jme3.niftygui.InputSystemJme.onMouseButtonEventQueued(InputSystemJme.java:234)
        at com.jme3.niftygui.InputSystemJme.forwardEvents(InputSystemJme.java:305)
        at de.lessvoid.nifty.Nifty.update(Nifty.java:368)
        at com.jme3.niftygui.InputSystemJme.endInput(InputSystemJme.java:115)
        at com.jme3.input.InputManager.processQueue(InputManager.java:859)
        at com.jme3.input.InputManager.update(InputManager.java:923)
        at com.jme3.app.LegacyApplication.update(LegacyApplication.java:774)
        at com.jme3.app.SimpleApplication.update(SimpleApplication.java:247)
        at com.jme3.system.lwjgl.LwjglAbstractDisplay.runLoop(LwjglAbstractDisplay.java:160)
        at com.jme3.system.lwjgl.LwjglDisplay.runLoop(LwjglDisplay.java:201)
        at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:242)
        at java.base/java.lang.Thread.run(Thread.java:829)
```

This may have been caused by trying to open the Windows start menu while adding an asset location.

The javadoc for `File.listFiles()` says:

> Returns null if this abstract pathname does not denote a directory, or if an I/O error occurs.

I should try to reproduce this on the Windows 10 laptop.

**Comment by stephengold** [comment]
On Windows 10 I produced a similar crash by trying to add "C:\users\sgold\SendTo" as an asset root:
```text
Oct 18, 2021 5:32:55 PM jme3utilities.nifty.BasicScreenController perform
SEVERE: Caught unexpected throwable:
java.lang.NullPointerException
	at maud.menu.EditorMenus.folderMap(EditorMenus.java:110)
	at maud.menu.BuildMenus.newAssetLocation(BuildMenus.java:496)
	at maud.action.NewAction.testForPrefixes(NewAction.java:207)
	at maud.action.NewAction.process(NewAction.java:172)
	at maud.action.EditorInputMode.onAction(EditorInputMode.java:180)
	at jme3utilities.nifty.BasicScreenController.perform(BasicScreenController.java:242)
	at jme3utilities.nifty.PopupMenu.onEvent(PopupMenu.java:219)
	at jme3utilities.nifty.PopupMenu.onEvent(PopupMenu.java:45)
```

**Comment by stephengold** [comment]
Fixed in "master" branch at 89d8147cf67b4a9a9205d904484751fb20e735cc,
